pkg/controller/nodes/subworkflow/launchplan/admin.go

```diff
@@ -246,12 +247,30 @@ func (a *adminLaunchPlanExecutor) syncItem(ctx context.Context, batch cache.Batc
 			continue
 		}
 
+		execData, err := a.adminClient.GetExecutionData(ctx, &admin.WorkflowExecutionGetDataRequest{
```

Comment:
For performance reasons, would it be worthwhile to wrap this call in an if block and only fetch execution data when [RawOutputPolicy](https://github.com/flyteorg/flytepropeller/blob/eadf07ba3ac2e54ee4aaf95a98c9941a5a005646/pkg/controller/config/config.go#L246) is set to inline?

Reply:
Discussed offline. We always need to call GetExecutionData()